01-node-tutorial/answers/week-1/.env

```diff
@@ -0,0 +1 @@
+process.env.MY_VAR='welcome to node'
```

Comment:
Out of curiosity, where you able to get this value? I think we don't use "process.env." inside of the env file but I'm not sure

Reply:
It actually did work, weirdly enough